_projects/2022/100470958/100470958.Rmd

```diff
@@ -0,0 +1,389 @@
+---
+title: "Satellite images defy economic data of authoritarian regimes"
+description: |
+  Highly customized dumbbell chart that compares official GDP against satellite-based estimation of GDP, across 103 countries.
+categories: "2022"
+author: Freddy Andrés Camargo
+date: "`r Sys.Date()`"
+citation: Camargo, 2023
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+```{r setup, include=FALSE, warning=FALSE}
+knitr::opts_chunk$set(warning = FALSE, message = FALSE) 
+```
+
+Satellite images of the brightness of the earth, also called nighttime lights data (NTL, from the US Air Force’s Defense Meteorological Satellite Program), have been used as a proxy for economic activity. Luis R. Martínez, professor at the Harris School of Public Policy at the University of Chicago, not only developed a model to calculate the GDP of every country based in the NTL data set, but he also found that countries with less political rights and civil liberties report overestimated GDP. The goal of this article is to rebuild a chart designed by The Economist to show the magnitude of this finding.
+
+![The Economist's chart](the_economist_gdp_growth_estimates.png "The Ecnomist's chart")
```

Comment:
Please append `{.external width="100%"} right after the closing parenthesis.

Reply:
done